types/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// TypeScript Version: 3.5
+
+import {Element, Properties, Node} from 'hast'
+
+declare function hastscript(
```

Comment:
Adding some TSDoc annotations can give additional context in typings aware editors.
Descriptions copied from _readme.md_

```suggestion
/**
 * DSL to create virtual hast trees for HTML or SVG
 *
 * @param selector Simple CSS selector
 * @param properties Map of properties
 * @param children (Lists of) child nodes
 */
declare function hastscript(
```